Examples/MagicWeatherSwiftUI/Magic Weather SwiftUI.xcworkspace/contents.xcworkspacedata

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Workspace
+   version = "1.0">
+   <FileRef
+      location = "group:Magic Weather SwiftUI.xcodeproj">
+   </FileRef>
+   <FileRef
+      location = "group:../../RevenueCat.xcodeproj">
```

Comment:
This is the critical line: the workspace points to the parent directory's project (so the project in the same repo).